frontend/src/modules/commentslist/components/CommentsList.css

```diff
@@ -23,3 +23,19 @@
     flex-flow: column wrap;
     width: 100%;
 }
+
+.team-comments-list .comment {
+    margin-left: 0;
+}
+
+.team-comments-list ul {
+    margin-left: .5em;
+}
+
+.no-team-comments > p {
+    border-bottom: 1px solid #5E6475;
+    color: #aaa;
+    font-style: italic;
+    line-height: 22px;
+    padding: 15px 10px;
+}
```

Comment:
This is specific to TeamComments, so it should go to `TeamComments.css`.

Reply:
`TeamComments` is now merged with `CommentsList` as per [suggested](https://github.com/mozilla/pontoon/pull/1561#discussion_r378891678). 
So there currently is no `TeamComments.css`, and if I add that file back in then we are at a similar place with 2 CSS modules being imported.

Reply:
They shouldn't be merged. There should be a `TeamComments` component, which renders the `CommentsList` component or the "No comments message".